types/oxon/oxobutton/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Oxobutton Q, (LoRaWAN)",
+  "name": "Oxobutton Q",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed